.env.tric

```diff
@@ -17,11 +17,17 @@ CLI_VERBOSITY=0
 # This will force the cli tool to use a specific project to run the tests.
 # You should use the `use` command usually.
 # TRIC_CURRENT_PROJECT=the-events-calendar
+# When you `here` at the site level, all selected targets via `use` will have a relative path set.
+# TRIC_CURRENT_PROJECT_RELATIVE_PATH=
```

Comment:
Since the `TRIC_CURRENT_PROJECT` hasn't been set, I don't think it makes sense to set the project's path. That env var isn't used until we've run `tric use`.

Reply:
Makes sense